src/best_of/generators/markdown_list.py

```diff
@@ -169,18 +169,24 @@ def generate_license_info(project: Dict, configuration: Dict) -> Tuple[str, int]
     if project.license:
         licenses_name = project.license
         licenses_warning = True
-        licenses_url = "https://tldrlegal.com/search?q=" + urllib.parse.quote(
-            project.license
-        )
-        license_metadata = get_license(licenses_name)
-
-        if license_metadata:
-            if license_metadata.name:
-                licenses_name = license_metadata.name
-            if license_metadata.url:
-                licenses_url = license_metadata.url
-            if "warning" in license_metadata:
-                licenses_warning = license_metadata.warning
+
+        # License can be a url
+        if licenses_name.startswith("http://") or licenses_name.startswith("https://"):
+            licenses_url = licenses_name
+            licenses_name = "Custom-url License"
```

Comment:
Since the length is very limited, I would propose to shorten it up to just `Custom`. Also, we should add a risk indicator here as well, similar how it is done with Unlicensed: 
![image](https://user-images.githubusercontent.com/2852129/106451047-491d8a00-6486-11eb-8d6c-d05d06978283.png)

Reply:
I changed the name as suggested. With regards to the risk warning, I think this is already handled in lines 191/192 as `licenses_warning` is still set to `True` in the custom case.

Reply:
Yep, that's right!